### ISSUE TITLE
Final report

### ISSUE THREAD

**Issue by ryanseys** [melari/RationalBlowsIssues#6]
Do eeeet.

**Comment by ryanseys** [comment]
https://drive.google.com/open?id=1y4MKPeNZ-ud1T-weM-SPZOUE4F2T-cr-wsfQvl_I0oE&authuser=0

**Comment by oharaandrew314** [comment]
- [ ] Introduction: Describe contributions of each member
- [x] System Requirements
- [x] Use Case Model 
  - [x] Use Case Descriptions
  - [x] Use Case Diagrams
- [ ] Analysis and Design
  - [ ] Describe Analysis and Design Phase from M1 and M2
  - [x] System Structure: Explain the role and responsibilities of different components in the final model
  - [ ]  Design Decisions: Discuss the choice of system architecture, concurrency issues, other design choices you have considered, role and responsibilities of different capsules/classes. Explain the use of UML-RT features such as encapsulation, multiplicity, static versus dynamic structure, different types of ports, capsules versus passive classes, inheritance, hierarchical state machines, etc
  - [ ] Illustrate the text with model printouts obtained from the tool (class diagrams, capsule
structure and state diagrams)
- [ ] Testing Strategy
  - [x] Briefly explain testing strategy
  - [ ] Unit Tests
  - [ ] Integration Tests
  - [ ] Illustrate the test cases with printouts of traces and/or sequence diagrams obtained by
running the model. Discuss how different sequence diagrams satisfy the original
requirements
- [x] Conclusion: Lessons Learned.  What was easy?  Hard?  Comment on Team Collaboration
- [ ] Submission
  - [ ] PDF of report
  - [ ] Copy of Project
  - [ ] Readme.txt with an ELI5 to hold TA's hand through running the program and injecting signals to perform tests
